src/MongoDB.Driver/MapReduceOptions.cs

```diff
@@ -177,12 +178,25 @@ public static MapReduceOutputOptions Inline
         /// <param name="sharded">Whether the output collection should be sharded.</param>
         /// <param name="nonAtomic">Whether the server should not lock the database for the duration of the merge.</param>
         /// <returns>A merge map-reduce output options.</returns>
+        [Obsolete("Use a Merge that does not have sharded and nonAtomic parameters instead.")]
```

Comment:
Slight rewording requested:

    [Obsolete("Use an overload of Merge that does not have sharded and nonAtomic parameters instead.")]

Reply:
done